anvil/config.py

```diff
@@ -114,6 +114,22 @@ def parse_sample_interval(self, interval: (int, type(None))):
         log.warning(f"Using user specified sample_interval: {interval}")
         return interval
 
+    def parse_bootstrap_n_samples(self, n_samples: (int, type(None))):
+        if n_samples is None:
+            return 1000  # default
```

Comment:
I *think* that it's better to not set defaults here and instead in the signature of the action which uses it, because then if you do something like `anvil-sample --help action` it will tell you the default value, whereas here it's completely obscured from user unless they open up the code

having said that the help messages are all messed up because reportengine can't interpret the numpy style docstring correctly..

Reply:
good point. Will change.